addons/gut/gut.gd

```diff
@@ -61,6 +61,7 @@ const LOG_LEVEL_ALL_ASSERTS = 2
 const WAITING_MESSAGE = '/# waiting #/'
 const PAUSE_MESSAGE = '/# Pausing.  Press continue button...#/'
 const COMPLETED = 'completed'
+const YIELD_FROM_OBJ = {}
```

Comment:
Since 4.0 compares dictionaries by value this should be something else so the bug does not show back up when being ported to 4.0.  A `Node` instance would work if we make this a `var` instead...which is fine with me.  Keep the all-caps so we know it should be constant.
```
var YIELD_FROM_OBJ = Node.new()
```

Reply:
I guess we don't care about leaking the node since gut is a singleton?

Reply:
I figured out you can use a class as a sentinel and then it can be used in places that require a constant expression. I'm not sure there is a better way to create a const sentinal with a unique identity